actors/evm/src/lib.rs

```diff
@@ -46,6 +46,8 @@ const EVM_MAX_RESERVED_METHOD: u64 = 1023;
 pub const NATIVE_METHOD_SIGNATURE: &str = "handle_filecoin_method(uint64,uint64,bytes)";
 pub const NATIVE_METHOD_SELECTOR: [u8; 4] = [0x86, 0x8e, 0x10, 0xc4];
 
+const EVM_WORD_SIZE: usize = 32;
```

Comment:
Diff is a bit noisy because this is now shared. I'll open a PR after this that moves some other random consts that are really EVM word size as well.